stacker/actions/build.py

```diff
@@ -229,31 +234,20 @@ def _launch_stack(self, stack, **kwargs):
                 logger.debug("Updating existing stack: %s", stack.fqn)
             except StackDidNotChange:
                 return DidNotChangeStatus()
+            except CancelExecution as e:
+                return CancelledStatus(reason=e.message)
 
         return new_status
 
-    def _generate_plan(self, tail=False):
-        plan_kwargs = {}
-        if tail:
```

Comment:
I'm not really sure what to do about the `--tail` flag. From playing with it on `master`, it doesn't seem to really work very well (it constantly gets overwritten by checkpoint'ing, see https://github.com/remind101/stacker/issues/386).

Since the DAG will likely go into 1.1, maybe we should pull `--tail` out? 

If we want to keep it in, we can make it work, but we should probably fix it in 1.0.x first.